stockflux-securities-master/src/components/form-view/form-controls/text-field/TextField.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './TextField.css';
+
+const TextField = ({ label, id, value, disabled, onChange }) => {
+  return (
+    <span className="input">
+      <label htmlFor={id}>{label}</label>
+      <input
+        className={classNames({ disabled: disabled })}
```

Comment:
You can change this to just `{ disabled }` and it should have the same effect, I think

Reply:
It would only work if I changed the value of `disabled` prop to a string `disabled`. For example:
```
        <TextField
          label="EXCHANGE"
          id="name"
          disabled={securityId && 'disabled'}
          value={state.exchange}
          onChange={event => dispatch(action.setExchange(event.target.value))}
        />
```

Changed that now.

Reply:
On Jonathons comment he is correct, a common abbreviation for classNames is cx. The cx function takes params as strings or objects, for objects it evaluates the value to true and uses the object prop name ast the string name of the class.

`<input className={cx({disabled})} />` will apply the class name disabled is the value of disabled is true.

I'd imagine with the new change you are theming based on `input[disabled]` or `input::disabled`

Reply:
I might have misunderstood Jonathan's initial comment - I thought he suggested not using `classnames` altogether and just use `className={disabled}`. Fixed now though to use `cx`.